core/chains/evm/client/errors.go

```diff
@@ -465,3 +465,15 @@ func NewSendErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction
 	}
 	return clienttypes.Unknown, err
 }
+
+// NewSendOnlyErrorReturnCode handles SendOnly nodes error codes. In that case, we don't assume there is another transaction that will be correctly
+// priced.
+func NewSendOnlyErrorReturnCode(err error) (clienttypes.SendTxReturnCode, error) {
```

Comment:
Would it make sense to call this `As` instead of `New`? And is it actually useful to return error when we just pass the input arg straight through?
```suggestion
func AsSendTxReturnCode(err error) clienttypes.SendTxReturnCode {
```

Reply:
If we rename this, let's rename the `NewSendErrorReturnCode` function as well. How about `SendErrorParse` & `SendOnlyErrorParse`, does that make more sense?
As for the error, it makes sense to remove it.